at_secondary/at_persistence_secondary_server/lib/src/model/at_meta_data.dart

```diff
@@ -85,6 +87,7 @@ class AtMetaData extends HiveObject {
     map[PUBLIC_DATA_SIGNATURE] = dataSignature;
     map[SHARED_KEY_ENCRYPTED] = sharedKeyEnc;
     map[SHARED_WITH_PUBLIC_KEY_CHECK_SUM] = pubKeyCS;
+    map['encoding'] = encoding;
```

Comment:
As per comment in the related at_tools PR - replace the string literal 'encoding' with a constant which is declared in the same place as SHARED_KEY_ENCRYPTED etc

Reply:
Replaced with constant.